serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc

```diff
@@ -0,0 +1,20 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-accessing-secrets-configmaps"]
+= Accessing secrets and ConfigMaps from Serverless functions
+include::modules/common-attributes.adoc[]
+:context: serverless-functions-secrets
+
+toc::[]
+
+From your functions, you can access data stored in `Secret` objects or in ConfigMaps. Such data can be either mounted as volumes or assigned to environment variables. You can configure this manually or with an interactive program.
```

Comment:
I'm not really clear on what "from your functions" means across these docs?
config maps
I'd call out that both ways are done via the Knative CLI but that one way offers prompts. Example where there are prompts when creating a function if you don't specify flags: https://docs.openshift.com/container-platform/4.7/serverless/functions/serverless-functions-getting-started.html#serverless-create-func-kn_serverless-functions-getting-started

Reply:
Changed the "from your functions" wording across the docs.

"I'd call out that both ways are done via the Knative CLI but that one way offers prompts" Only one is done via the Knative CLI, the other one is by direct editing of the YAML configuration file. If you meant the two different ways where the first way is by just running `kn func config` and the other is running, say, `kn func volumes add`, they are both interactive, the difference between them is how much you need to navigate once inside the interactive utility.